flybywire-aircraft-a320-neo/html_ui/Pages/VLivery/Liveries/Printer/Printer.js

```diff
@@ -13,7 +13,7 @@ class LiveryPrinter extends TemplateElement {
         const url = document.getElementsByTagName("livery-printer-element")[0].getAttribute("url");
         this.index = parseInt(url.substring(url.length - 1));
         if (this.index == 0) {
-            localStorage.setItem("pages", "[]");
+            NXDataStore.set("pages", "[]");
```

Comment:
**question:** the use of either `localStorage` or `NXDataStore` here is a bit sus. Why do we need that kind of persistence if we only access this state here, in this class ?

If it is needed, why was it changed to use `NXDataStore` ?

I'm aware this is not the scope of the PR, but since it ended up changing it, I think we could remove that if there is no use for it.

Reply:
This is indeed very sus. The reason it was implemented is because there's effectively two instances of the printer "paper" (one one the printer itself and one one the pedestal after clicking on it). Maybe we can sync those as well via events or find a way to eliminate the need of two instances. But I'd prefer having that in a separate PR. For now I can revert to localStorage again, I mainly did it for consistency as I didn't see any other direct usage of localStorage in the code.

Reply:
sessionStorage also works, and is erased by a trip through the main menu (localStorage is not).

Reply:
Actually, now that I think of it. Didn't we find out `xyzStorage` is very slow in Coherent ?

Reply:
I'll give sessionstorage a go. I think speed is not that much of an issue as the refresh interval of this is very low (afaik every 500 or even 1000ms)

Reply:
I've managed to remove persistence entirely.